### PR TITLE
fix: explicitly set required GITHUB_TOKEN permissions

### LINKED ISSUE #393: Signin token expired
```
Error:  ExecError: ExecError: (1) Generating ephemeral keys...
Retrieving signed certificate...
Non-interactive mode detected, using device flow.
Error: signing [renovate/renovate:31.13]: getting signer: getting key from Fulcio: retrieving cert: error obtaining token: expired_token
main.go:46: error during command execution: signing [renovate/renovate:31.13]: getting signer: getting key from Fulcio: retrieving cert: error obtaining token: expired_token
```

### PR DESCRIPTION
**Pull request by JamieMagee**
See https://github.com/renovatebot/docker-renovate/pull/244

Fixes #393

### CODE REVIEW

**Review by viceice — APPROVED** [review]

**Comment by JamieMagee** [comment]
@viceice I don't have maintainer permissions in this repo 😭 I can't merge this PR

**Comment by viceice** [comment]
I've waited for ci checks. 🤗

**Pull request merged**
